Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm

```diff
@@ -394,16 +394,16 @@
         return "AXCodeStyleGroup"_s;
 
     using namespace HTMLNames;
-    const auto& tag = tagName();
-    if (tag == kbdTag)
+    const auto& elementName = this->elementName();
```

Comment:
This probably shouldn't be const auto& anymore since the return type is uint16_t rather than an const AtomString& like tagName was.